templates/github/.github/workflows/scripts/before_install.sh.j2

```diff
@@ -167,12 +167,12 @@ then
 fi
 
 sed -i -e 's/DEBUG = False/DEBUG = True/' pulpcore/pulpcore/app/settings.py
-
-{% if not tasking_allow_async_unsafe -%}
+{%- if not tasking_allow_async_unsafe %}
 # Patch DJANGO_ALLOW_ASYNC_UNSAFE out of the pulpcore tasking_system
 # Don't let it fail. Be opportunistic.
-sed -i -e '/DJANGO_ALLOW_ASYNC_UNSAFE/d' ../pulpcore/pulpcore/tasking/entrypoint.py || true
-{% endif -%}
+sed -i -e '/DJANGO_ALLOW_ASYNC_UNSAFE/d' pulpcore/pulpcore/tasking/entrypoint.py || true
+{%- endif %}
```

Comment:
I just now realized, that i picked the best place for this by accident. :fireworks: ... and still got it wrong. :facepalm: